lumispy/components/convolved_gaussian_exponential.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019-2025 The LumiSpy developers
+#
+# This file is part of LumiSpy.
+#
+# LumiSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the license, or
+# (at your option) any later version.
+#
+# LumiSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with LumiSpy. If not, see <https://www.gnu.org/licenses/#GPL>.
+
+import numpy as np
+
+import hyperspy.api as hs
+
+sigma2fwhm = 2 * np.sqrt(2 * np.log(2))
+
+
+class ConvGaussExp(hs.model.components1D.Expression):
+    r"""Analytical convolution of Gaussian instrument response function and
+    exponential decay function.
+
+    .. math::
+
+        f(x) = \frac{h}{2} \cdot \exp\left[(x - t_0 - \frac{\sigma^2}{2\tau})\tau^{-1}\right]
```

Comment:
Add a reference where the math comes from? 
For example, https://math.stackexchange.com/questions/685041/convolute-exponential-with-a-gaussian/872561#872561 has good pointers?

Reply:
So far, it is based on https://zenodo.org/records/6384277 - but we are checking the math again with mathematica. As perspective we will at least add a component with double exponential as well.

Reply:
Out of curiosity, are you using mathematica because sympy can't do it or just haven't tried with sympy?

Reply:
A student already did the calculations in mathematica and I did not have the time yet to copy that in sympy :-)